ext/xmlwriter/php_xmlwriter.stub.php

```diff
@@ -6,85 +6,85 @@ function xmlwriter_open_uri(string $uri): XMLWriter|false {}
 
 function xmlwriter_open_memory(): XMLWriter|false {}
 
-function xmlwriter_set_indent(XMLWriter $xmlwriter, bool $indent): bool {}
+function xmlwriter_set_indent(XMLWriter $writer, bool $enable): bool {}
 
-function xmlwriter_set_indent_string(XMLWriter $xmlwriter, string $indentString): bool {}
+function xmlwriter_set_indent_string(XMLWriter $writer, string $indentation): bool {}
 
-function xmlwriter_start_comment(XMLWriter $xmlwriter): bool {}
+function xmlwriter_start_comment(XMLWriter $writer): bool {}
 
-function xmlwriter_end_comment(XMLWriter $xmlwriter): bool {}
+function xmlwriter_end_comment(XMLWriter $writer): bool {}
 
-function xmlwriter_start_attribute(XMLWriter $xmlwriter, string $name): bool {}
+function xmlwriter_start_attribute(XMLWriter $writer, string $name): bool {}
 
-function xmlwriter_end_attribute(XMLWriter $xmlwriter): bool {}
+function xmlwriter_end_attribute(XMLWriter $writer): bool {}
 
-function xmlwriter_write_attribute(XMLWriter $xmlwriter, string $name, string $value): bool {}
+function xmlwriter_write_attribute(XMLWriter $writer, string $name, string $content): bool {}
```

Comment:
Why `$value` is not the preferred name? I'd choose "attribute value" rather than "attribute content".